### PR TITLE
git sync --all: push tags

### LINKED ISSUE #464: git sync --all: should push tags to the remote
When running `git sync --all`, it should also run `git push --tags` to push tags to the remote.

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo @allewun 

resolves #464

### CODE REVIEW

**Comment by kevgo** [comment]
This seems to break a bunch of other tests.

**Comment by charlierudolph** [comment]
@kevgo @allewun This is green now

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Git Town Release Notes
 
+## Unreleased
+
+* `git sync --all`: push tags
```
Comment (kevgo) [inline RELEASE_NOTES.md]: Are we going back to documenting unreleased features in the respective PRs? (I always liked it that way)

**Comment by kevgo** [comment]
:ship:

File: RELEASE_NOTES.md (hunk 1 above)
Comment (charlierudolph) [inline RELEASE_NOTES.md]: Yeah. Read more about changelogs when I added it to other repos and think this is nice. Also makes the actual releases easier.

Comment (charlierudolph) [inline RELEASE_NOTES.md]: Also lets people know whats already in it in the case that we don't release for a while. Though hopefully this serves as a reminder to release often

**Pull request closed without merging**